Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -603,8 +609,8 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
             path.translate(clipOffset);
 
         clipPath = path;
-    } else if (const auto& renderBox = dynamicDowncast<RenderBox>(regionRenderer)) {
-        auto borderShape = BorderShape::shapeForBorderRect(renderBox->style(), renderBox->borderBoxRect());
+    } else if ((regionRendererBox = dynamicDowncast<RenderBox>(regionRenderer))) {
```

Comment:
You don't need the extra parentheses right?
```suggestion
    } else if (regionRendererBox = dynamicDowncast<RenderBox>(regionRenderer)) {
```

Reply:
Without them there's a build error "using the result of an assignment as a condition without parentheses"